ext/field.go

```diff
@@ -0,0 +1,17 @@
+package ext
+
+import (
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+)
+
+// LogError add error event record for the Span
```

Comment:
`LogError sets the error=true tag on the Span and logs err as an "error" event.`